src/components/Callout/Callout.tsx

```diff
@@ -17,8 +18,14 @@ export const Callout = ({
       variation="filled"
       colorTheme={warning ? 'warning' : 'info'}
       backgroundColor={backgroundColor}
+      hasIcon={false}
     >
-      <View>{children}</View>
+      <Flex>
+        <div role="img" aria-label="info" className="amplify-message__icon">
+          <IconInfo aria-hidden={true} />
```

Comment:
I think we should to account for the warning variation as well

Reply:
I've added in labeling to account for the variation however currently only the Info icon is being used regardless of the message type

Reply:
There is a warning variation that is used for Callout, like here: https://docs.amplify.aws/swift/build-a-backend/add-aws-services/analytics/set-up-analytics/#initialize-amplify-analytics

Reply:
Although I just checked and we don't have an IconWarning so it would have to be added.

Reply:
Happy to add it! Had just been attempting to maintain the current appearance on the prod site.

Reply:
Added the IconWarning and adjusted the logic to account for the variation